torchreid/models/efficient_net_pytcv.py

```diff
@@ -341,7 +341,8 @@ def __init__(self,
                         tf_mode=tf_mode))
                 in_channels = out_channels
             self.features.add_module("stage{}".format(i + 1), stage)
-        activation = activation if self.loss == 'softmax' else lambda: nn.PReLU(init=0.25)
+        self.loss = self.loss.split(',') if ',' in self.loss else self.loss
+        activation = activation if 'softmax' in self.loss else lambda: nn.PReLU(init=0.25)
```

Comment:
@supersoob @harimkang @goodsong81  I found some suspicious code, self.loss is defined as 'softmax,asl' in multihead task, so I confirmed activation function is always defined as swish(default), not PReLU. Could you review this line?